core/src/tile/tileWorker.cpp

```diff
@@ -15,27 +15,34 @@ void TileWorker::abort() {
     m_aborted = true;
 }
 
-void TileWorker::processTileData(std::unique_ptr<WorkerData> _workerData,
+void TileWorker::processTileData(std::unique_ptr<TileTask> _task,
                                  const std::vector<std::unique_ptr<Style>>& _styles,
                                  const View& _view) {
 
-    m_workerData = std::move(_workerData);
+    m_task = std::move(_task);
```

Comment:
Ohh, I thought a move assignment operator was needed for this!! and hence I had added one.